Lesson 36_HW01/src/rdx/usersSlice.tsx

```diff
@@ -0,0 +1,39 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit'
+ 
+
+export interface fetchType {
+  data: any
+}
+
+const initialState: fetchType = {
+  data: [],
+}
+
+export const fetchUsersMiddleware = createAsyncThunk(
```

Comment:
```suggestion
export const fetchUsers = createAsyncThunk(
```
дивна назва
Middleware це точно не про thunk